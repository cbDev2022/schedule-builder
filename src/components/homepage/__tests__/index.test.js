import React from "react";
import { render, cleanup } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';
import Homepage from '..';

afterEach(cleanup);

describe('Homepage component', () => {
    //renders Homepage test
    // First Test
    it('renders', () => {
        render(<Homepage />);
    });

    // Second Test
    it('matches snapshot DOM node structure', () => {
        // render Homepage
        const { asFragment } = render(<Homepage />);
        expect(asFragment()).toMatchSnapshot();
    })

})