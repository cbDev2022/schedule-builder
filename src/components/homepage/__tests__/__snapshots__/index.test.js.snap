// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="homepage"
    >
      This is the homepage
    </h1>
  </section>
</DocumentFragment>
`;
